### PR TITLE
prepare 1.34

### PR DESCRIPTION
**Pull request by Simon-Laux**
and remove update 90days reminder from desktop,
because core already has one anyway

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,21 @@
 
 ### Changed
 
+### Fixed
+
+
+<a id="1.34.0"></a>
+
+## [1.34.0] - 2022-11-24 - Testrelease
```
Comment (dotlambda) [inline CHANGELOG.md:14]:
```suggestion
## [1.34.0] - 2022-11-26
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,21 @@
 
 ### Changed
 
+### Fixed
+
+
+<a id="1.34.0"></a>
```
Comment (dotlambda) [inline CHANGELOG.md:12]:
```suggestion
```

File: src/renderer/deviceMessages.ts [hunk 3]
```diff
@@ -1,76 +1,20 @@
...
-Especially contact us if you encounter a specific bug, where desktop feels like it is half frozen, where it reacts to clicks, but not to events (unread badge in chat list won't disappear, message state stays spinning even though the message is already sent out successfully): https://github.com/deltachat/deltachat-desktop/issues/2992
-
-Full changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md` // no anchor link because this is a test version
-  )
+    'changelog-version-1.34.0-version0',
+    `What's new in 1.34.0?
+
+    ⚡️ Faster UI thanks to jsonrpc
+    🧹 Clear chat history
+    🔍 Search for Messages in chat
```
Comment (dotlambda) [inline src/renderer/deviceMessages.ts:11]:
```suggestion
    🔍 Search for messages in chat
```

File: src/renderer/deviceMessages.ts [hunk 4]
```diff
@@ -1,76 +1,20 @@
...
+    ⏬ jump down button
+    🤗 Friendlier contact lists: Ordered by last seen and contacts seen within 10 minutes are marked by a dot 🟢
```
Comment (dotlambda) [inline src/renderer/deviceMessages.ts:13]:
```suggestion
    🤗 Friendlier contact lists: ordered by last seen and contacts seen within 10 minutes are marked by a dot 🟢
```
Why aren't the green dots an additional bullet point?

File: src/renderer/deviceMessages.ts [hunk 5]
```diff
@@ -1,76 +1,20 @@
...
+    🎛️ Tray icon is now enabled by default
+    🔔 when receiving multiple Notifications at once, they will be grouped into one single Notification
```
Comment (dotlambda) [inline src/renderer/deviceMessages.ts:15]:
```suggestion
    🔔 when receiving multiple notifications at once, they will be grouped into one single notification
```

File: src/renderer/deviceMessages.ts [hunk 6]
```diff
@@ -1,76 +1,20 @@
...
-
-Thanks for testing Delta Chat, please report bugs on GitHub.
-Especially contact us if you encounter a specific bug, where desktop feels like it is half frozen, where it reacts to clicks, but not to events (unread badge in chat list won't disappear, message state stays spinning even though the message is already sent out successfully): https://github.com/deltachat/deltachat-desktop/issues/2992
-
-Full changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md` // no anchor link because this is a test version
-  )
+    'changelog-version-1.34.0-version0',
+    `What's new in 1.34.0?
+
+    ⚡️ Faster UI thanks to jsonrpc
```
Comment (r10s) [inline src/renderer/deviceMessages.ts:9]:
```suggestion
    ⚡️ Smooth rendering. Much snappier scrolling. The new version is easy on your patience and is the result of a refactoring that took more than a year (for our tech-savvy friends: we are now using JSON-RPC under the hood and threw away lots of old, annoying code :)  
```

File: CHANGELOG.md (hunk 1 above)
Comment (Simon-Laux) [inline CHANGELOG.md:14]:
```suggestion
## [1.34.0] - 2022-11-27
```

File: CHANGELOG.md (hunk 2 above)
Comment (dotlambda) [reply CHANGELOG.md:12]: I'm pretty sure it works without.

Comment (dotlambda) [reply CHANGELOG.md:12]: Oh but the default anchor is annoyingly long.

**Review by dotlambda — APPROVED** [review]

**Review by r10s — APPROVED** [review]

File: src/renderer/deviceMessages.ts [hunk 7]
```diff
@@ -1,76 +1,20 @@
...
+    🧹 Clear chat history
+    🔍 Search for messages in chat
+    ⏬ jump down button
+    🤗 Friendlier contact lists: ordered by last seen and contacts seen within 10 minutes are marked by a dot 🟢
+    🎛️ Tray icon is now enabled by default
+    🔔 when receiving multiple notifications at once, they will be grouped into one single notification
+    😀 right click to add sticker to the sticker selector
+    ✨ Many smaller bug fixes and general improvements
+
+Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1.34.0`)
```
Comment (Simon-Laux) [inline src/renderer/deviceMessages.ts:19]:
```suggestion
Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1340---2022-11-27`)
```

File: CHANGELOG.md (hunk 2 above)
Comment (Simon-Laux) [reply CHANGELOG.md:12]: I think the "right" solution would be having a short-link on a deltachat domain

Comment (Simon-Laux) [reply CHANGELOG.md:12]:
```suggestion
<a id="1_34_0"></a>
```

File: src/renderer/deviceMessages.ts (hunk 7 above)
Comment (Simon-Laux) [reply src/renderer/deviceMessages.ts:19]:
```suggestion
Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_34_0`)
```

File: CHANGELOG.md (hunk 2 above)
Comment (Simon-Laux) [reply CHANGELOG.md]: actually I think github is just weird, dots in anchors work as good as underscores, but I changed to underscores now anyway

**Pull request merged**